pkg/auth/dependency/webapp/url_provider.go

```diff
@@ -86,38 +84,39 @@ type AuthenticateURLProvider struct {
 }
 
 func (p *AuthenticateURLProvider) AuthenticateURL(options AuthenticateURLOptions) (httputil.Result, error) {
-	authnURI := p.Endpoints.AuthenticateEndpointURL()
-	q := map[string]string{
-		"redirect_uri": options.RedirectURI,
-		"client_id":    options.ClientID,
+	intent := &Intent{
+		RedirectURI: options.RedirectURI,
+		KeepState:   false,
+		UILocales:   options.UILocales,
+		Intent:      interactionintents.NewIntentLogin(),
 	}
-	if options.Prompt != "" {
-		q["prompt"] = options.Prompt
-	}
-	if options.UILocales != "" {
-		q["ui_locales"] = options.UILocales
+	inputer := func() (interface{}, error) {
+		return nil, nil
 	}
+
+	intent.RedirectURI = options.RedirectURI
+	intent.UILocales = options.UILocales
```

Comment:
These two lines seem redundant. We have set the two fields when intent is constructed at line 87.

Reply:
oops forgot to delete them when moving them up...